gocd/leapmicro.target.gocd.yaml

```diff
@@ -56,3 +56,57 @@ pipelines:
         tasks:
           - script: osc -A https://api.opensuse.org release --target-project=openSUSE:Leap:Micro:6.0:ToTest --target-repository=product -r product openSUSE:Leap:Micro:6.0 000productcompose
 
+  LeapMicro61.RelPkgs:
+    group: Leap
+    lock_behavior: unlockWhenFinished
+    timer:
+      spec: 0 10 * ? * *
+      only_on_changes: false
+    materials:
+      git:
+        git: https://github.com/openSUSE/openSUSE-release-tools.git
+    environment_variables:
+      OSC_CONFIG: /home/go/config/oscrc-staging-bot
+    stages:
+    - Create.Release.Packages:
+        approval: manual
+        resources:
+        - repo-checker
+        tasks:
+          - script: ./pkglistgen.py -A https://api.opensuse.org update_and_solve -p openSUSE:Leap:Micro:6.1 -s target --only-release-packages
+
+  LeapMicro61.Package.Lists:
```

Comment:
I think this would not be needed with __all__ see https://build.opensuse.org/requests/1221435